rewrite-java/src/main/java/org/openrewrite/java/internal/template/BlockStatementTemplateGenerator.java

```diff
@@ -193,14 +202,6 @@ private boolean isTemplateStopComment(Comment comment) {
         return js;
     }
 
-    private void template(Cursor cursor, J prior, StringBuilder before, StringBuilder after, J insertionPoint, JavaCoordinates.Mode mode) {
-        if (contextSensitive) {
-            contextTemplate(cursor, prior, before, after, insertionPoint, mode);
-        } else {
-            contextFreeTemplate(cursor, prior, before, after);
-        }
-    }
```

Comment:
Inlined above for simplicity, as this was only used once and gave a confusion indirection with unused arguments.